src/components/atoms/Input/index.tsx

```diff
@@ -15,9 +15,11 @@ export type InputProps = {
   value?: string;
   errorMessage?: string;
   errorIcon?: boolean;
+  errorPadding?: 'small' | 'normal' | 'large';
```

Comment:
In which cases we're going to need it? Can't we standardize it to "normal" for all inputs?